Tests/UnitTests/Purchasing/Purchases/PurchasesTests.swift

```diff
@@ -0,0 +1,855 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  Created by RevenueCat.
+//
+
+import Nimble
+import StoreKit
+import XCTest
+
+@testable import RevenueCat
+
+class PurchasesTests: BasePurchasesTests {
```

Comment:
⚠️ this file was split starting from #1609, but looks like it got re-added here!